.github/ISSUE_TEMPLATE/newrepo.yml

```diff
@@ -0,0 +1,166 @@
+        How about after merging, on main?
+        Are additional tests needed?
+      placeholder: |
+        Provide a high level overview -- exact details aren't essential
+    validations:
+      required: true
+  - type: textarea
+    id: dependencies
+    attributes:
+      label: Dependencies
+      description: |
+        Does this depend on other Egeria repos, or them on this? ie for building, testing?
+      placeholder: |
+        No dependencies directly on other repos, All egeria dependencies are via maven/gradle.
+    validations:
+      required: true
+  - type: textarea
+    id: justification
+    attributes:
+      label: Justification
+      description: |
+        Why do we need another repo? Were alternatives looked at using existing repos?
+      placeholder: |
+        Add justification here
+    validations:
+      required: true
+  - type: textarea
+    id: assumptions
+    attributes:
+      label: Assumptions
+      description: |
+        Are all of the following assumptions true? If not please explain why
+          - DCO is required
+          - Apache 2.0 / CC-BY-4.0 licensed
+          - Issue tracking, projects enabled, wikis/discussions disabled
+          - Security scanning will be enabled
+          - code scanning will be enabled
+          - User docs are in egeria-docs, some technical detailed docs/designs in this repo
+          - initial CODEOWNERS will list repository owner primarily
+          - Dependabot dependency updates will be set up
+          - nominated owner will be responsible for managing the repo content
+          - repo is public with same group/permissions as the main Egeria repo
+          - If using maven/gradle, snapshots will be published to oss.sonatype.org and releases to maven central
+          - If using containers, they will be published to quay.io & docker.io & built for arm64 + amd64 with same release versions as other artifacts
+          - If using maven, versioning should follow 1.1 / 1.1-SNAPSHOT convention. For javascript etc use semver, ie 1.1.1 / 1.1.1-rc.0
+          - Follows egeria branching-strategy ie branch just before release
+          - Security policy aligned with egeria - common mailing list for reporting vulnarabilities
+      placeholder: |
+        Yes - all true
+    validations:
+      required: true
+  - type: textarea
+    id: extra
+    attributes:
+      label: Additional Information
+      description: |
+        Any other information?
+      placeholder: |
+        add any other information here
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+        label: Work Plan
+        description: | 
+          DO NOT FILL THIS IN NOW
+          
+          This is a checklist to guide the work needed to complete getting the repository and code into an appropriate state for ongoing development. 
+                
+          Where these items may not be undertaken soon after creation, an issue should be opened to track.
```

Comment:
Maybe we should document these "DO NOT FILL IN NOW" considerations (or some of these) more advanced / potentially optional checks in egeria-docs and link from the issue template  and (perhaps) not include in the issue template. Being in Egeria docs would mean one place this info is documented and could be searched for.

Reply:
I'm unsure.. A simplification perhaps, but I think it's good to be extra clear at the point someone is making the submission. It's easy to miss the detailed docs

The short-term challenge is that those docs don't exist yet (feel free if you'd like to contribute) so a checklist/template was one way to get things moving quickly.

Reply:
Adding to https://github.com/odpi/egeria/issues/6298